### PR TITLE
Bump osb-broker-lib to 0.0.4

### PR DESCRIPTION
**Pull request by LiliC**
Closes https://github.com/pmorie/osb-starter-pack/issues/74

### CODE REVIEW

File: pkg/broker/logic.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ type BusinessLogic struct {
 	instances map[string]*exampleInstance
 }
 
-var _ broker.BusinessLogic = &BusinessLogic{}
+var _ broker.Interface = &BusinessLogic{}
```
Comment (LiliC) [inline pkg/broker/logic.go]: Not sure if we should rename `BusinessLogic` struct as well, didn't find the naming that bad, but can change?

Comment (carolynvs) [reply pkg/broker/logic.go]: FWIW, I thought the struct name was fine. It's a placeholder name anyway in the starter pack, whereas the other value (broker.Interface) is a permanent name in the library.

File: Gopkg.toml [hunk 2]
```diff
@@ -35,5 +35,5 @@
 
 [[constraint]]
   name = "github.com/pmorie/osb-broker-lib"
-  version = "0.0.2"
+  version = "0.0.3"
```
Comment (carolynvs) [inline Gopkg.toml]: There are breaking changes  on 0.0.3 (originating identity) that are fixed in master. Can we cut an new release and then bump this too? Doesn't have to be in this PR, seeing this just reminded me! 😁

Comment (LiliC) [reply Gopkg.toml]: I know, had that in the back of my mind as well. Wanted to get the ball rolling and get things ready, for when @pmorie cuts a release of the lib. ;) 

We can just wait for that to happen and will update this PR, as this change is not that urgent.

**Comment by LiliC** [comment]
@carolynvs Updated to the new 0.0.4 release! Feel free to merge if it still looks good to you. ;)

**Pull request merged**